### PR TITLE
Filter by action in Redux Devtools

### PR DESCRIPTION
**Pull request by Mokshit06**
Rel #9482 

image

### CODE REVIEW

**Comment by Mokshit06** [comment]
The current style of this panel is similar to the `Elements` Panel, but sure, padding above the input?

**Comment by jasonLaster** [comment]
On second thought - lets add a follow up issue for @jonbell-lot23

File: src/ui/components/SecondaryToolbox/ReduxDevTools.tsx [hunk 1]
```diff
@@ -37,9 +38,12 @@ export const ReduxDevToolsPanel = () => {
   const reduxAnnotations: ReduxActionAnnotation[] = useMemo(() => {
     const annotations = annotationsStatus === "resolved" ? parsedAnnotations : [];
     return annotations.filter(
-      annotation => focusWindow && isPointInRegion(annotation.point, focusWindow)
+      annotation =>
```
Comment (markerikson) [inline src/ui/components/SecondaryToolbox/ReduxDevTools.tsx:41]: :hand: Probably need to check if there even is a non-empty `searchValue` before we do that comparison

Comment (Mokshit06) [reply src/ui/components/SecondaryToolbox/ReduxDevTools.tsx:41]: I don't think that's required with `String#match` as it treats the second string as a regex, but on second thought `String#includes` is probably better in this case

**Pull request merged**